src/rust/plugin-work-queue/migrations/1640037910_plugin_registry_migration.sql

```diff
@@ -0,0 +1,80 @@
+CREATE SCHEMA plugin_work_queue;
+
+CREATE TYPE plugin_work_queue.status_t AS ENUM ('enqueued', 'failed', 'processed');
+
+CREATE FUNCTION plugin_work_queue.megabytes(bytes integer) RETURNS integer AS
```

Comment:
We should add an `IF NOT EXISTS` clause to this (and to _every_ object that honors that clause).

Reply:
There is no IF NOT EXISTS for CREATE FUNCTION but I can add OR REPLACE. Given that a migration should never run twice I don't think it should come up but it seems fine to put in defensively.

Reply:
Ah, right, `OR REPLACE`.

Yeah, I just reflexively add idempotency clauses everywhere when writing SQL. It's also handy when you're iterating locally on a file of DDL; you can just edit and reload.